### PR TITLE
Tweak travis a bit

### PR DESCRIPTION
**Pull request by steelbrain**
Beep Boop

[Review on Reviewable](https://reviewable.io/reviews/atomlinter/atom-linter/50)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 language: objective-c
 
+sudo: false
```
Comment (SpainTrain) [inline .travis.yml]: I think this is the default?

Comment (steelbrain) [inline .travis.yml]: I think this enables the new container based builds

Comment (SpainTrain) [inline .travis.yml]: depends when it was first added to travis (http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure)

explicit is of course good all the same =-)

Comment (Arcanemagus) [inline .travis.yml]: @SpainTrain All of the projects @steelbrain _just_ enabled Travis on for me a week or two ago needed that explicitly stated. I'm pretty sure it still isn't the default until they get `sudo` working inside the containers.

**Pull request merged**